src/Symfony/Component/Messenger/Bridge/AmazonSqs/Middleware/MessageDeduplicationAwareInterface.php

```diff
@@ -20,5 +20,5 @@ interface MessageDeduplicationAwareInterface
      * The max length of MessageDeduplicationId is 128 characters.
      * Valid values: alphanumeric characters and punctuation (!"#$%&'()*+,-./:;<=>?@[]^_`{|}~).
      */
-    public function getMessageDeduplicationId(): string|null;
+    public function getMessageDeduplicationId(): ?string;
```

Comment:
@keradus is there a rule for that? :)

Reply:
https://github.com/symfony/symfony/pull/52174